react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/FontEditor.tsx

```diff
@@ -168,7 +190,11 @@ const CustomFont: React.FunctionComponent<Props> = ({
         // TODO: treat errors on delete
         return
       }
-      showToast('Font family saved succesfully.')
+      showToast(
+        intl.formatMessage({
+          id: 'admin/pages.editor.styles.edit.font-family.add-font-success',
```

Comment:
Ops, just saw the messages on this file, maybe you could change these `intl.formatMessage` to use the value returned from `defineMessages` as well